apps/omg_eth/test/support/dev_geth.ex

```diff
@@ -33,7 +33,7 @@ defmodule OMG.Eth.DevGeth do
     keystore = Path.join([homedir, "/geth/keystore"])
     datadir = Path.join([homedir, "/geth"])
     :ok = File.write!("/tmp/geth-blank-password", "")
-    geth = ~s(geth --syncmode 'fast' --miner.gastarget 7500000 \
```

Comment:
This isn't the only occurrence of this `syncmode: fast` `syncmode: full` thing. `docker-compose.yml` is another one, let's fix consistently everywhere (not sure if only these 2)

Reply:
Ooh indeed. I found only one more as well. Fixed!